src/operator/every.ts

```diff
@@ -66,12 +63,13 @@ class EverySubscriber<T, R> extends Subscriber<T> {
   }
 
   protected _next(value: T): void {
-    const result = tryCatch(this.predicate).call(this.thisArg || this, value, this.index++, this.source);
-
-    if (result === errorObject) {
-      this.destination.error(result.e);
-    } else if (!result) {
-      this.notifyComplete(false);
+    try {
+        const result = this.predicate.call(this.thisArg || this, value, this.index++, this.source);
+        if (!result) {
```

Comment:
you don't want to `notifyComplete` instead of this try/catch block. If it errors in the completion handler, it will send the error down the error path, which isn't the correct behavior.